src/com/crawljax/core/Crawler.java

```diff
@@ -188,8 +188,7 @@ private void tryToFireEvent(StateVertex targetState, StateVertex curState, Event
 			plugins.runOnRevisitStatePlugins(context, curState);
 
 		} else {
-			throw new StateUnreachableException(targetState, "couldn't fire eventable "
-			        + clickable);
+			throw new StateUnreachableException(targetState, "couldn't fire eventable " + clickable.toString());
```

Comment:
Ok, just wanted to make sure it was an intentional and recognized change. Not just something that accidentally snuck in.